_topic_map.yml

```diff
@@ -620,18 +620,29 @@ Topics:
         File: ruby_on_rails
       - Name: Setting Up a Nexus Mirror
         File: maven_tutorial
-  - Name: Opening a Remote Shell to Containers
-    File: ssh_environment
-  - Name: Templates
-    File: templates
-  - Name: Service Accounts
-    File: service_accounts
   - Name: Builds
-    File: builds
-  - Name: Managing Images
-    File: managing_images
-  - Name: Quotas and Limit Ranges
-    File: compute_resources
+    Dir: builds
+    Topics:
+      - Name: How Builds Work
+        File: index
+      - Name: Basic Build Operations
+        File: basic_build_operations
+      - Name: Build Inputs
+        File: build_inputs
+      - Name: Build Output
+        File: build_output
```

Comment:
@bparees Yeah, your points are some of the things I wrestled with on these titles. Basically what Ashley and Ben have mentioned here are a lot of the reasons I felt it was the least of the evils.